index.html

```diff
@@ -0,0 +1,209 @@
+                                <li>My page</li>
+                                <li>Login</li>
+                            </ul>
+                        </div>
+                        <div class="relNav">
+                            <div>
+                                <a href="index.html"><ul class="navle">
+                                    <li><img class="logoN" src="img/plug_icon.png" alt=""></li>
+                                    <li><h1>MyPlug</h1></li>
+                                </ul></a>
+                            </div>
+                            <div>
+                                <ul class="navRig">
+                                    <li><h2><a href="index.html" class="menuli">Home</a></h2></li>
+                                    <li><h2><a href="about.html" class="menuli">About</a></h2></li>
+                                    <li><h2><a href="#ph" class="menuli">Intro</a></h2></li>
+                                    <li><h2><a href="#We_do" class="menuli">Create</a></h2></li>
+                                    <li><h2><a href="#tea" class="menuli">Team</a></h2></li>
+                                    <li><h2><a href="#pat" class="menuli">Partners</a></h2></li>
+                                </ul>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="mobileNav">
+                        <span class="closeMenu" onclick="closeMobileMenu()">X</span>
+                        <ul class="mobile">
+                            <li><h2><a href="index.html" onclick="closeMobileMenu()">Home</a></h2></li>
+                            <li><h2><a href="about.html" onclick="closeMobileMenu()">About</a></h2></li>
+                            <li><h2><a href="#ph" onclick="closeMobileMenu()">Intro</a></h2></li>
+                            <li><h2><a href="#We_do" onclick="closeMobileMenu()">We do</a></h2></li>
+                            <li><h2><a href="#tea" onclick="closeMobileMenu()">Team</a></h2></li>
+                            <li><h2><a href="#pat" onclick="closeMobileMenu()">Partners</a></h2></li>
+                        </ul>
+                    </div>
+                </nav>
+            </div>
+            <div class="phrases" id="ph">
+                <h3 class="phrasesh2">"We create with tech"</h3>
+                <h1 class="phraseh1">CREATIVITY AND <br> INNOVATIVENESS AT ITS <br> BEST LEVEL</h1>
+            </div>
+            <div class="headParaDiv">
+                <p class="headPara">Learning gives creativity, creativity 
+                    leads to thinking, thinking providdes 
+                    knowledge, knowledge makes you great <br>
+                    -Abdul Kalam
+                </p>
+            </div>
+            <div class="headLoc">
+                <h2 class="headL">Plot 256, PO BOX 24, SRT 42</h2>
+                <p class="headLocw">@nagauru block 25.</p>
+            </div>
```

Comment:
- [OPTIONAL] It would be better to choose a different color for your titles and text because this part doesn't have enough contrast with the background. If you fix them it would improve the readability of your site. 😉 

![font-color](https://user-images.githubusercontent.com/79876959/142710907-f57109f0-9105-477d-af73-2f12483eac60.PNG)